src/release.sh

```diff
@@ -13,6 +13,13 @@ NPM_INFO="undefined"
 
 # instead of using -v use -n to check for an empty strings
 # -v is not working well on bash 3.2 on osx
+PATTERNFLY_DEPS="undefined"
+if [[ -f package-lock.json ]];
+then
+  LINES=`npm list --silent --depth=0 --production | grep @patternfly -i | sed -r "s/^(.{0})(.{4})/\1/" | tr "\n" "," | sed -r "s/,/\",\"/g"` 
```

Comment:
Is there an objection to using `-E` instead of `-r`? MacOS problems 😄

Reply:
oh :D I have not considered those heretics :dagger:. Give me a second.